core/src/i18n/en.yaml

```
@@ -977,6 +977,8 @@ events:
     source: Source
     type: Type
   title: Events
+extensibility:
+  error: Something went wrong. Please confirm the extension configuration is correct.
```

```suggestion
  error: Your extension configuration is incorrect.
```
Taking into account our [UI guidelines}(https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/04-ui-elements.md#error-messages) I'd go with something like this + maybe we can add a link that shows the user how to fix the configuration?